src/components/expression/expression-renderer.js

```diff
@@ -178,6 +178,42 @@ genotet.ExpressionRenderer.Profile = function(params) {
   this.color = params.color != null ? params.color : null;
 };
 
+/**
+ * Zoom status object storing the status of expression matrix.
+ * @param {!{
```

Comment:
My bad here. I double-checked the docs and see that record types and typedef (usually seen as {{...}}) is non-nullable by default. We can get rid of the exclamation mark.